.github/workflows/publish-dry-run.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          toolchain: stable
+          toolchain: 1.85.0
```

Comment:
Might be best to use `rust-toolchain.toml` file to avoid having to manually update all workflows, then we can remove `toolchain` from the workflows altogether: https://github.com/actions-rust-lang/setup-rust-toolchain?tab=readme-ov-file#inputs